packages/google_sign_in/google_sign_in_web/lib/src/gis_client.dart

```diff
@@ -265,6 +282,8 @@ class GisSdkClient {
   // This function returns the currently signed-in [GoogleSignInUserData].
   //
   // It'll do a request to the People API (if needed).
+  //
+  // @Deprecated
```

Comment:
Why is a private method deprecated?

Reply:
I added these because I don't want to forget about removing these when actually removing the `signIn` method, but this should probably be an issue + proper TODO.

I'll create the issue right away and link it properly.

Reply:
Linked these breadcrumbs to this issue:

* https://github.com/flutter/flutter/issues/137727